src/service/alerts/alert.rs

```diff
@@ -794,6 +795,9 @@ async fn process_dest_template(
         )
     };
 
+    // Shorten the alert url
+    let alert_url = ShortUrl::new(None).shorten(&alert_url).await;
```

Comment:
i saw we use `shorturl::new(None)` every where, and never saw `shoturl::new(Some(xx))`, maybe we can reduce the parameter, or, we can directly call `shorturl::shorten(xxx)`

Reply:
refactored to use functions.